src/lib.rs

```diff
@@ -121,14 +123,20 @@ pub mod pipeline {
             let mut in_agg = false;
             let mut pre_agg: Vec<Box<dyn operator::UnaryPreAggOperator>> = Vec::new();
             let mut post_agg: Vec<Box<dyn operator::AggregateOperator>> = Vec::new();
-            let mut op_iter = query.operators.into_iter().peekable();
+            let mut op_deque = query.operators.into_iter().collect::<VecDeque<_>>();
             let mut has_errors = false;
-            while let Some(op) = op_iter.next() {
+            while let Some(op) = op_deque.pop_front() {
                 match op {
                     Operator::RenderedAlias(rendered_alias) => {
                         // TODO: create a new QueryContainer here and parse the templated string
                         // -> expecting only Inline operators?
                         // let inner_query = QueryContainer::new(rendered_alias.value, ???);
+                        let (_span, operators) =
+                            operator_list(Span::new(CompleteStr(&rendered_alias.value)))
```

Comment:
lets change this error to something a bit more helpful -- I think we could hit this during development of aliases if there's a typo in the alias? Or is that validated elsewhere

Reply:
Yeah this will fail if you're creating an alias and had typos / not parsable things.

Added a more useful error message containing the rendered alias (the err returned is a nom error and not very readable).